src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+@use '../utils/variables' as *;
+
+.card {
+  box-sizing: border-box;
+  border: 1px solid #{$gray-elements-color};
+  border-radius: 5px;
+
+  width: 200px;
+  height: 408px;
```

Comment:
Don't apply a fixed width or height to blocks. In this case, it's explicitly stated that the card has a width of 200px, so that's acceptable, but let the content determine the height of the card.